README.md

```diff
@@ -427,6 +428,36 @@ testList "Setup & teardown 5" [
 ]
 ```
 
+### Theory tests
+
+- `testTheory : string -> seq<'a> -> ('a -> 'b) -> Test`
+
+The test theory takes a name and a sequence of cases to test against.
+The `'a` parameter will be inferred to the *sequence type*, such as `string -> seq<int> -> (int -> 'b) -> Test`.
+
+Example:
+
+```fsharp
+testList "theory testing" [
+  testTheory "odd numbers" [1; 3; 5] <| fun x ->
+    Expect.isTrue (x % 2 = 1) "should be odd"
+]
+```
+
+- `testTheoryWithResult: string -> seq<'a * 'b> -> ('a * 'b -> 'c) -> Test`
```

Comment:
`testTheoryWithResult` didn't make the cut. 
The example could still be good to keep though.

Reply:
@ratsclub I noticed you just merged. Can you hold on release until the readme is updated?